schema.graphql

```diff
@@ -169,6 +169,28 @@ type FpmmParticipation @entity {
   timeout: BigInt
 }
 
+enum TradeType {
+  Buy
+  Sell
+}
+
+type FpmmTrade @entity {
+  id: ID!
+  fpmm: FixedProductMarketMaker!
+
+  # lifted from fpmm
+  title: String
+  collateralToken: Bytes!
+
+  type: TradeType!
+  creator: String!
+  creationTimestamp: BigInt!
+  amount: BigInt!
```

Comment:
Maybe `collateralAmount` would be a better name?

Reply:
Will add your suggestions! Thanks @cag !